src/SprykerSdk/Zed/Benchmark/BenchmarkDependencyProvider.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\Benchmark;
+
+use GuzzleHttp\Cookie\CookieJar;
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerSdk\Zed\Benchmark\Dependency\Service\BenchmarkToUtilEncodingServiceBridge;
+
+/**
+ * @method \SprykerSdk\Zed\Benchmark\BenchmarkConfig getConfig()
+ */
+class BenchmarkDependencyProvider extends AbstractBundleDependencyProvider
+{
+    public const SERVICE_UTIL_ENCODING = 'SERVICE_UTIL_ENCODING';
+    public const CLIENT_PERFORMANCE_AUDIT = 'CLIENT_PERFORMANCE_AUDIT';
+    public const COOKIE_JAR = 'COOKIE_JAR';
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideBusinessLayerDependencies(Container $container): Container
+    {
+        $container = $this->addPerformanceAuditClient($container);
+        $container = $this->addCookieJar($container);
+        $container = $this->addUtilEncodingService($container);
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addPerformanceAuditClient(Container $container): Container
+    {
+        $container->set(static::CLIENT_PERFORMANCE_AUDIT, function (Container $container) {
+            return $container->getLocator()->performanceAudit()->client();
```

Comment:
No bridge required here?

Reply:
It's a leftover from the copypasting and renaming. So this should be a client from this module. IMO bridge is not required in this case.